tvm_utility/CMakeLists.txt

```diff
@@ -0,0 +1,87 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(tvm_utility)
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
+
+find_package(autoware_build_flags REQUIRED)
+find_package(OpenCV REQUIRED)
```

Comment:
Please add:

```
find_package(Boost REQUIRED COMPONENTS system)
```

Reply:
Done

Reply:
@JWhitleyWork do you remember why was boost required?

Reply:
I don't, actually. And looking at it now, it seems that it's only a `test_depend` and none of the tests actually use it. It can probably be removed.